src/Xamarin.Android.Build.Tasks/Tasks/CompileToDalvik.cs

```diff
@@ -97,23 +100,23 @@ protected override string GenerateCommandLineCommands ()
 				cmd.AppendSwitchIfNotNull("-Xmx", JavaMaximumHeapSize);
 
 				cmd.AppendSwitchIfNotNull ("-jar ", Path.Combine (DxJarPath));
-				cmd.AppendSwitch ("--no-strict");
 			}
 
-			cmd.AppendSwitch ("--dex");
+			cmd.AppendSwitch (DxExtraArguments);
 
 			if (MultiDexEnabled) {
 				cmd.AppendSwitch ("--multi-dex");
 				cmd.AppendSwitchIfNotNull ("--main-dex-list=", MultiDexMainDexListFile);
 			}
-			cmd.AppendSwitchIfNotNull ("--output=", Path.GetDirectoryName (ClassesOutputDirectory));
+			cmd.AppendSwitchIfNotNull ("--output ", Path.GetDirectoryName (ClassesOutputDirectory));
```

Comment:
can we get away without the `=` on this? Any idea how this will effect builds on windows?

Reply:
Just tried a quick command run on Windows; java -jar c:\path\to\dx.jar --dex --no-strict --output foobar "path\to\mono.android.jar" worked fine.